node/tests/initialization_test.rs

```diff
@@ -193,7 +193,7 @@ fn started_without_explicit_chain_parameter_runs_finely() {
     );
 
     node.wait_for_log("UIGateway bound", Some(5000));
-    assert_eq!(node.kill().unwrap().code().unwrap(), 0);
```

Comment:
"started_without_explicit_chain_parameter_runs_smoothly" and "started_without_explicit_chain_parameter_runs_finely" should be the same above.